pkg/virt-launcher/virtwrap/network/podinterface_test.go

```diff
@@ -93,9 +93,6 @@ var _ = Describe("Pod Network", func() {
 		ctrl = gomock.NewController(GinkgoT())
 		mockNetwork = NewMockNetworkHandler(ctrl)
 		podnic = podNICImpl{cacheFactory: cacheFactory, handler: mockNetwork}
-		podNICFactory = func(cacheFactory cache.InterfaceCacheFactory) podNIC {
```

Comment:
It should be removed from the `AfterEach` as well.

Reply:
Done